src/app/routes/ChainMempool.tsx

```diff
@@ -0,0 +1,144 @@
+import { Buffer } from "buffer"; // eslint-disable-line unicorn/prefer-node-protocol
```

Comment:
im getting an error in the mempool, cant access it 
![image](https://github.com/iotaledger/wasp-dashboard/assets/3624944/92e16d0d-1f51-46f7-8347-aefb08f6e346)